src/common/k8s-api/api-kube.ts

```diff
@@ -17,4 +17,4 @@ export const apiKube = isClusterPageContext()
       "Host": window.location.host,
     },
   })
-  : undefined;
+  : undefined as never;
```

Comment:
I see many places with `never` in this PR. Why do we use `never` for example here?

Reply:
I use `never` here to remove the `| undefined` from the type of `const apiKube`. I do this to reduce churn since all of these globals will be removed in the future anyway I didn't think it worth it to change all the uses in this PR.